### ISSUE TITLE
Add check for SELinux impact to curation process

### ISSUE THREAD

**Issue by jason-callaway** [fedoraredteam/elem#64]
Need to start tracking if SELinux impacts the efficacy of exploits.

**Comment by kevensen** [comment]
@jason-callaway Agreed.  Let me run this by you.  When the **staging** information is added to an exploit, the **SELinux** is set as permissive or enforcing.

You and I had discussed altering the score to also reflect SELinux's impact.  For example, a 000008 if it had to be permissive or 000009 if the exploit worked as enforcing.

Does the combination of this information satisfy this issue?

**Comment by kevensen** [comment]
Come to think of it, in any case, we need to document and detail our rationale for scoring.
